Applications/WebPortal/WebApp/library/source/components/index.ts

```diff
@@ -9,4 +9,5 @@ export * from './integer_input_box';
 export * from './modal';
 export * from './money_input_box';
 export * from './number_input';
+export * from './single_security_input';
```

Comment:
Just like you don't name it ```single_number_input```, similarly don't name it ```single_security_input```. The name ```security_input``` is sufficient.